krator/Cargo.toml

```diff
@@ -35,8 +35,8 @@ async-trait = "0.1"
 anyhow = "1.0"
 tokio = { version = "1.0", features = ["fs", "macros", "signal"] }
 tokio-stream = { version = "0.1", features = ['sync'] }
-kube = { version = "0.60", default-features = false }
-kube-runtime = { version = "0.60", default-features = false }
+kube = { version = "0.64", features = ['client'] }
```

Comment:
Since `default-features` is enabled here, won't this force `client` and `native-tls`?

Reply:
Good catch, I've submitted a commit that fixes that (hopefully)